features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioBLE.cpp

```diff
@@ -41,6 +41,10 @@
 /*! WSF handler ID */
 wsfHandlerId_t stack_handler_id;
 
+/* WSF heap allocation */
+uint8_t *SystemHeapStart;
```

Comment:
Could these be made static? (and `stack_handler_id` above as well)

Reply:
no, they need to be provided for wsf